core/src/main.cpp

```diff
@@ -198,7 +466,7 @@ auto print_configuration_template() -> void
 
         "requests": {{
             "threads": 3,
-            "max_rbuffer_size_in_bytes": 8388608,
+            "max_size_of_body_in_bytes": 8388608,
```

Comment:
Should this be `max_size_of_request_body_in_bytes`?

Reply:
Changed to suggested name.